skorch/helper.py

```diff
@@ -78,3 +79,13 @@ def __repr__(self):
     @property
     def shape(self):
         return (self._len,)
+
+
+def filtered_optimizer(optimizer):
```

Comment:
As discussed in #261, this should probably have a different name. Besides, I think docstring can be more descriptive. Right now without checking the code it's not obvious whether `optimizer` argument should be a class or an instantiated object.